spring-cloud-aws-messaging/src/main/java/org/springframework/cloud/aws/messaging/core/QueueMessageChannel.java

```diff
@@ -68,20 +71,53 @@ protected boolean sendInternal(Message<?> message, long timeout) {
 		return true;
 	}
 
-	private Map<String, MessageAttributeValue> getContentTypeMessageAttributes(Message<?> message) {
-		Map<String, MessageAttributeValue> messageAttributes = new HashMap<>(1);
-		Object mimeType = message.getHeaders().get(MessageHeaders.CONTENT_TYPE);
-		if (mimeType != null) {
-			if (mimeType instanceof MimeType) {
-				messageAttributes.put(MessageHeaders.CONTENT_TYPE, new MessageAttributeValue().withDataType("String").withStringValue(mimeType.toString()));
-			} else if (mimeType instanceof String) {
-				messageAttributes.put(MessageHeaders.CONTENT_TYPE, new MessageAttributeValue().withDataType("String").withStringValue((String) mimeType));
+	private Map<String, MessageAttributeValue> getMessageAttributes(Message<?> message) {
```

Comment:
Actually this method uses the logger which is an instance field. So I'll leave it as is.